client/homebrew/editor/editor.jsx

```diff
@@ -52,7 +60,7 @@ const Editor = createClass({
 	getInitialState : function() {
 		return {
 			editorTheme : this.props.editorTheme,
-			view        : 'text' //'text', 'style', 'meta'
+			view        : 'text' //'text', 'style', 'meta', 'snip'
```

Comment:
personally i'd just name this tab `snippets` rather than `snip`-- it seems like an unnecessary abbreviation in something that'll get minified later.